elements/pfe-accordion/BaseAccordionHeader.scss

```diff
@@ -0,0 +1,113 @@
+:host {
+  --pf-c-accordion__toggle--Color: var(--pf-global--Color--100, #151515);
+  --pf-c-accordion__toggle--BackgroundColor: transparent;
+  --pf-c-accordion__toggle--after--BackgroundColor: transparent;
+  --pf-c-accordion__toggle--PaddingTop: var(--pf-global--spacer--sm, 0.5rem);
+  --pf-c-accordion__toggle--PaddingRight: var(--pf-global--spacer--md, 1rem);
+  --pf-c-accordion__toggle--PaddingBottom: var(--pf-global--spacer--sm, 0.5rem);
+  --pf-c-accordion__toggle--PaddingLeft: var(--pf-global--spacer--md, 1rem);
+  --pf-c-accordion__toggle--FontSize: var(--pf-global--FontSize--lg, 1rem);
+  --pf-c-accordion__toggle--FontFamily: var(--pf-global--FontFamily--redhat-updated--heading--sans-serif, "RedHatDisplayUpdated", helvetica, arial, sans-serif);
```

Comment:
PF's using RedHatTextUpdated for their toggle headings. Sorry if I didn't flag this before!

Reply:
No worries at all, I missed this as well.  Thank you!